tests/functions.php

```diff
@@ -0,0 +1,50 @@
+<?php
+require '../functions.php';
+use PHPUnit\Framework\TestCase;
+
+class functions extends TestCase {
+    public function testIsStringDisplayAlbums() {
+        $albums = [
+            [
+                'image-link' => 'blskdjfldksjf',
+                'name' => 'test',
+                'release-year' => '1999',
+                'artist' => 'blah blah',
+                'genre' => 'music',
+                'record-label' => 'pow'
+            ],
+        ];
+        $case = displayAlbums($albums);
+        $this->assertIsString($case);
+    }
+
+    public function testMalformedDisplayAlbums()
+    {
+        $input = 3;
+        $this->expectException(TypeError::class);
+        displayAlbums($input);
+    }
+
+    public function testFailureDisplayAlbums() {
```

Comment:
This test fails when I run it. The idea of a failure test is to make sure your code has a way of handling an error. E.g. an empty array being passed into your function.